course_discovery/services.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from edx_rest_api_client.client import EdxRestApiClient
+
+
+logger = logging.getLogger()
+
+
+class CatalogApiService(object):
```

Comment:
@MatthewPiatetsky We already have a `CatalogApiService` at https://github.com/edx/ecommerce-scripts/blob/master/edx_sailthru/sailthru_content/services/catalog_api_service.py
I know what you want here is super simple. I would argue the one in `edx_sailthru` can use this service here. Is there a good reason or consideration to have these CatalogApiService independent?